apps/dashboard/src/components/RightPanel/SizePositionSection.tsx

```diff
@@ -17,53 +19,121 @@ export function SizePositionSection({
       <div className="grid grid-cols-2 gap-2">
         <Input
           onChange={(value) => {
-            updateElement({
-              ...selectedElement,
-              x: value,
+            selectedElements.forEach((selectedElement) => {
+              updateElement({
```

Comment:
Could we try to have a single state update for all selected elements at once? That would also make a better ctrl+z/Z experience

Reply:
I think if we edit updateElement like that, we won't be able to change mixed values with up/down arrows, as we can't get and change unique value of every element without looping. So like, if the elements will have x position of 300 and 400, we can't change their values to +1/-1 at the same time.

Reply:
Maybe we can use updateElements function when we don't have mixed values. The arguments will be array of elements and object that contain state to update. And for the mixed values we can create another function, that will accept array of elements and object with key as an id of element and value will be the object that contain state to update.